### PR TITLE
Fix module init

### PR DESCRIPTION
**Pull request by zhangchiqing**
See inline comments

### CODE REVIEW

File: cmd/execution_builder.go [hunk 1]
```diff
@@ -287,23 +283,18 @@ func (exeNode *ExecutionNode) LoadGCPBlockDataUploader(
 			exeNode.collector,
 		)
 
-		exeNode.events = storage.NewEvents(node.Metrics.Cache, node.DB)
-		exeNode.commits = storage.NewCommits(node.Metrics.Cache, node.DB)
-		exeNode.computationResultUploadStatus = storage.NewComputationResultUploadStatus(node.DB)
-		exeNode.transactions = storage.NewTransactions(node.Metrics.Cache, node.DB)
-		exeNode.collections = storage.NewCollections(node.DB, exeNode.transactions)
-		exeNode.txResults = storage.NewTransactionResults(node.Metrics.Cache, node.DB, exeNode.exeConf.transactionResultsCacheSize)
+		computationResultUploadStatus := storage.NewComputationResultUploadStatus(node.DB)
```
Comment (zhangchiqing) [inline cmd/execution_builder.go:286]: should just be a local module, if not being used by others.

**Review by vishalchangrani — APPROVED** [review]

File: cmd/execution_builder.go (hunk 1 above)
Comment (Kay-Zee) [reply cmd/execution_builder.go:286]: could even move into the initializer, to further imply it's locality, similar to how we're doing the `NewDownloader`

**Review by Kay-Zee — APPROVED** [review]
This is assuming that everything is properly initialized in the `initStorage function`

File: cmd/execution_builder.go [hunk 2]
```diff
@@ -107,44 +107,40 @@ type ExecutionNode struct {
...
-	ledgerStorage                 *ledger.Ledger
-	events                        *storage.Events
-	serviceEvents                 *storage.ServiceEvents
-	txResults                     *storage.TransactionResults
-	results                       *storage.ExecutionResults
-	myReceipts                    *storage.MyExecutionReceipts
-	computationResultUploadStatus *storage.ComputationResultUploadStatus
-	commits                       *storage.Commits
-	collections                   *storage.Collections
-	transactions                  *storage.Transactions
```
[the remark this answers is not in the document]
Comment (zhangchiqing) [reply cmd/execution_builder.go:123]: yeah, would be more clear to hide white space

image

image

**Comment by zhangchiqing** [comment]
bors merge

**Pull request merged**